RedditSharp/Things/Comment.cs

```diff
@@ -94,15 +94,40 @@ public Comment PopulateComments(IEnumerator<Thing> things)
             return this;
         }
 
-        private void ParseComments(JToken data, Thing sender)
+        private void ParseComments(JToken data, Thing sender, IWebAgent agent)
         {
             // Parse sub comments
             var replies = data["data"]["replies"];
             var subComments = new List<Comment>();
             if (replies != null && replies.Count() > 0)
             {
                 foreach (var comment in replies["data"]["children"])
-                    subComments.Add(new Comment(WebAgent, comment, sender));
+                {
+                    if (comment.Value<string>("kind") != "more")
+                    {
+                        subComments.Add(new Comment(WebAgent, comment, sender));
+                    }
+                    else
+                    {
+                        /*
+                         * If a comment is in a deep comment chain, reddit sends a single `more` object with name `t1__` in place of the 
+                         * comment's replies. This is the equivalent of seeing a 'Continue this thread' link on the HTML site.
+                         * We get the actual replies by sending another request for this same Comment on it's own
+                        */
+                        if (comment["data"]["name"].ToString() == "t1__")
+                        {
+                            Task<Comment> getFullSelfTask = new Reddit(agent).GetCommentAsync(this.Subreddit, this.Id, this.LinkId);
+                            getFullSelfTask.Wait();
```

Comment:
Change signature to async and use await instead.

Reply:
Should probably make this optional as well so a thread doesn't chew up a ton of resources on load with requests.

Reply:
Agreed making it optional is better. This is called in the constructor, so do we want a new Constructor signature like so:
`public Comment(IWebAgent agent, JToken json, Thing sender, bool resolveDeepNestedThreads = false) : base(agent, json) {`
that way it's not a breaking change?